packages/theme-data/README.md

```diff
@@ -47,8 +47,23 @@ console.log(lightGrayMediumDensityTheme);
 // }
 ```
 
+### Access theme data as Material UI Theme
```

Comment:
I have not reviewed the code in detail yet, but there is one change I would like to see. I think the Material UI theming code should be in a separate package, not part of `packages/theme-data`. The theme  data project is for defining the basic theming data, and there should be separate projects for each of the particular implementations of the themes, such as material UI. I suggest that this code go in `packages/theme-material-ui`.